migrations/20190327101332-add-status-to-notifications.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn("notifications", "status", {
```

Comment:
Hey yves!
How many differents status can we have?
if it's only 2 : read or not read a boolean type is more appropriate.
if it's less than 4 you can save it in a (enum)[http://docs.sequelizejs.com/class/lib/data-types.js~ENUM.html].
It will help to increase query speed in future..